packages/core/src/browser/browser.ts

```diff
@@ -30,7 +30,9 @@ export const isWebKit = (userAgent.indexOf('AppleWebKit') >= 0);
 export const isChrome = (userAgent.indexOf('Chrome') >= 0);
 export const isSafari = (userAgent.indexOf('Chrome') === -1) && (userAgent.indexOf('Safari') >= 0);
 export const isIPad = (userAgent.indexOf('iPad') >= 0);
-export const isNative = typeof (window as any).process !== 'undefined';
+export const isNative = typeof (window as any).process !== 'undefined'; // tslint:disable-line:no-any
```

Comment:
Is this really needed? Does using 'any' without the tslint exception now fails the build>